plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -323,11 +323,9 @@ func getFitPredicateFunctions(names sets.String, args PluginFactoryArgs) (map[st
 	}
 
 	// Always include mandatory fit predicates.
-	for name, mandatory := range mandatoryFitPredicates {
-		if mandatory {
-			if factory, found := fitPredicateMap[name]; found {
-				predicates[name] = factory(args)
-			}
+	for _, name := range mandatoryFitPredicates.List() {
```

Comment:
Please do not use `List()` here. It first copies all the items of the map (set) into a slice and then sorts the items and returns the slice. We don't need a sorted list. Instead, just use:
`for name, _ := range mandatoryFitPredicates {`